app/templates/main/index.html

```diff
@@ -1,8 +1,11 @@
 {% extends 'layouts/base.html' %}
-
+{# Some special functionality here: current_user represents the current user. is_authticated is false for an AnonymousUser
```

Comment:
Yes I couldn't quite separate the commits such that they were actually discrete to every pull request.